packages/termcap/build.sh

```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=https://www.gnu.org/software/termutils/
+TERMUX_PKG_DESCRIPTION="Termcap is a library and data base that enables programs to use display terminals in a terminal-independent manner."
+TERMUX_PKG_DEPENDS="termcap"
+TERMUX_PKG_VERSION=1.3.1
+TERMUX_PKG_SRCURL=https://ftp.gnu.org/gnu/termcap/termcap-${TERMUX_PKG_VERSION}.tar.gz
```

Comment:
You can use http link instead of ftp `http://mirrors.kernel.org/gnu/termcap/termcap-${TERMUX_PKG_VERSION}.tar.gz`

Reply:
Just to clarify, it is served over HTTP(S), indicated by the protocol prefix.

Would you like the SRCURL changed to kernel.org and re-submitted?

Reply:
Sure, all other gnu packages use `http://mirrors.kernel.org/gnu`

Reply:
Good to go. Thanks for requesting to match the same mirror for other GNU tools. Consistency is key!